src/cli.go

```diff
@@ -59,7 +60,8 @@ func newAPIClient(ua *userEndpointAuth, cache bool) *sourcegraph.Client {
 
 	var transport http.RoundTripper
 	if cache {
-		transport = http.RoundTripper(httpcache.NewTransport(diskcache.New("/tmp/srclib-cache")))
+		tempdir := path.Join(os.TempDir(), "srclib-cache")
```

Comment:
I'm unsure if the `srclib-cache` is actually reused, but we can ignore that for now and just preserve the behavior as before (while making it cross platform).

So using `os.TempDir()` is fine.

But you should be using `filepath.Join` here, not `path.Join`, since you're joining a system filepath.